docs/source/concepts/cl_columns.rst

```diff
@@ -46,12 +64,54 @@ Contains a boolean value, with ``True`` indicating that the sample is likely to
 .. _suggested_label:
 ``suggested_label``
 ---------------
-Contains the suggested label for the sample. If the sample is not a label issue (``is_label_issue == False``), the suggested label will be empty. For samples with label issues, the suggested label is computed by Cleanlab studio.
+Contains the suggested label for the sample. If the sample is not a label issue (``is_label_issue`` marked as ``False``), the suggested label will be empty. For samples with label issues, the suggested label is computed by Cleanlab studio.
 
 Outliers
 ========
 
 .. _is_outlier:
 ``is_outlier``
 -------
-Contains a boolean value, with ``True`` indicating that the sample is likely to be an outlier.
\ No newline at end of file
+Contains a boolean value, with ``True`` indicating that the sample is likely to be an outlier.
+
+Ambiguous
+=========
+
+.. _is_ambiguous:
+``is_ambiguous``
+----------
+Contains a boolean value, with ``True`` indicating that the sample is likely to be ambiguous. Ambiguous samples are those that do not obviously belong to a single class.
+
+High Confidence
+===============
+
+.. _is_high_confidence:
+``is_high_confidence``
+---------------
+Contains a boolean value, with ``True`` indicating high confidence, which means that the given label of the sample is likely to be correct, so the sample can be safely used in downstream tasks.
+
+Uncertain
+=========
+
+.. _is_uncertain:
+``is_uncertain``
+-----------
+Contains a boolean value, with ``True`` indicating uncertain. Uncertain data specifically means data that are likely to be mislabeled (they will have ``is_label_issue`` marked ``True``), but that Cleanlab is uncertain about what their correct label should be. Such data would benefit from human review.
+
+Near Duplicates
+===============
+
+.. _is_near_duplicate:
+``is_near_duplicate``
+----------------
+Contains a boolean value, with ``True`` indicating that the sample is likely to be a near duplicate of another sample. Near duplicates are two or more examples in a dataset that are extremely similar to each other, relative to the rest of the dataset. The examples flagged with this issue may be exactly duplicated, or lie atypically close together when represented as vectors (i.e. feature embeddings).
```

Comment:
```suggestion
Contains a boolean value, with ``True`` indicating that the sample is likely to be a near duplicate of another sample. Near duplicates are two or more examples in a dataset that are extremely similar (or identical) to each other, relative to the rest of the dataset.
```

Suggestion, omit the implementation detail that we're looking for data points that are close in embedding space. The other explanations don't include such detail.